lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs

```diff
@@ -525,6 +526,33 @@ spec = do
             r <- request @ApiWallet ctx (Link.postWallet @'Shelley) headers payload
             verify r expectations
 
+    -- https://github.com/input-output-hk/cardano-wallet/issues/2120
+    it "WALLETS_CREATE_10 - Address pool gap isn't limited to Word8" $ \ctx -> do
```

Comment:
Would you mind adjusting also test `WALLETS_CREATE_08 - address_pool_gap` to have assertion for `addrPoolMax`... somehow it is missing it.

Reply:
I have a feeling this was omitted precisely of that bug...

Reply:
@piotr-iohk I believe this one is enough then, even and I can delete the other?